protocol/triple/triple-tool/protoc-gen-go-triple/main.go

```diff
@@ -51,19 +53,41 @@ func main() {
 		os.Exit(1)
 	}
 
-	protogen.Options{}.Run(
+	var flags flag.FlagSet
+	useOld := flags.Bool("useOldVersion", false, "print the version and exit")
+	old_triple.RequireUnimplemented = flags.Bool("require_unimplemented_servers", true, "set to false to match legacy behavior")
+
+	protogen.Options{
+		ParamFunc: flags.Set,
+	}.Run(
 		func(plugin *protogen.Plugin) error {
 			plugin.SupportedFeatures = uint64(pluginpb.CodeGeneratorResponse_FEATURE_PROTO3_OPTIONAL)
-			for _, file := range plugin.Files {
-				if file.Generate {
-					tripleGo, err := generator.ProcessProtoFile(file.Proto)
-					if err != nil {
-						return err
-					}
-					return generator.GenTripleFile(tripleGo)
-				}
+			if *useOld {
+				return genOldTriple(plugin)
```

Comment:
Will the generated `oldTriple` sources be put in the same place and given the same package name as in earlier versions?

Reply:
Keep the same as before